zlib.cabal

```diff
@@ -101,7 +101,10 @@ library
   build-tools:     hsc2hs >= 0.67 && < 0.69
     -- GHC 7 ships hsc2hs-0.67
 
-  includes:        zlib.h
+  includes:        zlib.h hs-zlib.h
+  install-includes: hs-zlib.h
```

Comment:
Could you please add a comment what's the deal with both `includes` and `install-includes`? I always forget the difference.